lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_codefreeze_prechecks.rb

```diff
@@ -64,6 +78,18 @@ def self.available_options
                                        description: 'Default branch of the repository',
                                        type: String,
                                        default_value: Fastlane::Helper::GitHelper::DEFAULT_GIT_BRANCH),
+          FastlaneCore::ConfigItem.new(key: :build_gradle_path,
+                                       description: 'Path to the build.gradle file',
+                                       type: String,
+                                       optional: true,
+                                       conflicting_options: [:version_properties_path]),
```

Comment:
IINM, this also conflicts with a developer using `project_root_folder`, no?

Reply:
Correct, along with `project_name` as well. Addressed in https://github.com/wordpress-mobile/release-toolkit/commit/4af6d4e2cb1be043e27ddcf32fcc6c145a6fd441